manifests/slave.pp

```diff
@@ -141,6 +141,14 @@
         before => Service['jenkins-slave'],
       }
     }
+    'RedHat': {
```

Comment:
Indeed.  EL < 7 does not and there are likely amazon linux variants still in use that are pre-systemd as well.